include/can/core/messages.hpp

```diff
@@ -124,7 +124,6 @@ struct AddLinearMoveRequest : BaseMessage<MessageId::add_move_request> {
     ticks duration;
     um_per_tick_sq acceleration;
     um_per_tick velocity;
-    uint8_t request_stop_condition;
```

Comment:
I think you accidentally removed this here? We'll want to be able to specify a stop condition for linear moves sometimes (i.e. for capacitance).